tests/dialects/test_stencil.py

```diff
@@ -459,35 +448,26 @@ def test_stencil_load_bounds():
 @pytest.mark.parametrize(
     "attr, dims",
     (
-        (
-            i32,
-            ArrayAttr(
-                [IntegerAttr[IntegerType](1, 64), IntegerAttr[IntegerType](2, 64)]
-            ),
-        ),
+        (i32, [64, 64]),
         (
             i64,
-            ArrayAttr(
-                [
-                    IntegerAttr[IntegerType](1, 32),
-                    IntegerAttr[IntegerType](2, 32),
-                    IntegerAttr[IntegerType](3, 32),
-                ]
-            ),
+            [
+                32,
+                32,
+                32,
+            ],
         ),
     ),
 )
 def test_stencil_temptype_constructor_with_ArrayAttr(
-    attr: IntegerType, dims: ArrayAttr[AnyIntegerAttr]
+    attr: IntegerType, dims: list[int]
 ):
     stencil_temptype = TempType(dims, attr)
 
     assert isinstance(stencil_temptype, TempType)
     assert stencil_temptype.element_type == attr
     assert stencil_temptype.get_num_dims() == len(dims)
-    assert stencil_temptype.get_shape() == [
-        list(dims.data)[dim].value.data for dim in range(len(dims))
-    ]
+    assert stencil_temptype.get_shape() == [list(dims)[dim] for dim in range(len(dims))]
```

Comment:
```suggestion
    assert stencil_temptype.get_shape() == dims
```
Am I missing something?

Reply:
:scream: